WordPress/src/main/java/org/wordpress/android/ui/media/MediaBrowserActivity.java

```diff
@@ -522,13 +525,23 @@ public void onUploadProgress(MediaModel media, float progress) {
         // TODO
     }
 
+    private void showMediaToastError(@StringRes int message, @Nullable String messageDetail) {
+        if (isFinishing())
+            return;
+        String errorMessage = getString(message);
+        if (!TextUtils.isEmpty(messageDetail)) {
+            errorMessage += ", " + messageDetail;
```

Comment:
This should probably be a colon instead of a comma - otherwise our error messages look odd:

`Media upload error occurred, Sorry, you have used your space allocation`.